tools/ci_build/github/linux/copy_strip_binary.sh

```diff
@@ -53,6 +53,7 @@ cp $SOURCE_DIR/README.md $BINARY_DIR/$ARTIFACT_NAME/README.md
 cp $SOURCE_DIR/docs/Privacy.md $BINARY_DIR/$ARTIFACT_NAME/Privacy.md
 cp $SOURCE_DIR/LICENSE $BINARY_DIR/$ARTIFACT_NAME/LICENSE
 cp $SOURCE_DIR/ThirdPartyNotices.txt $BINARY_DIR/$ARTIFACT_NAME/ThirdPartyNotices.txt
+cp $SOURCE_DIR/tools/ci_build/github/azure-pipelines/licenses/ROCmNotices.txt $BINARY_DIR/$ARTIFACT_NAME/ROCmNotices.txt
```

Comment:
is this supposed to be done for non-ROCm packages?

Reply:
I was having trouble propagating ROCmNotices.txt and I was trying to mimic Privacy.md to figure it out. It seems this script is not invoked during training package build. So I reverted it and awaiting validation.

Reply:
if it's for building a python package, it should be here: https://github.com/microsoft/onnxruntime/blob/master/cmake/onnxruntime_python.cmake#L303

Reply:
nm, looks like you found it :)